ds-controller-board/driver/driver.go

```diff
@@ -52,10 +56,25 @@ func (drv *ControllerBoardDriver) Initialize(lc logger.LoggingClient, asyncCh ch
 
 	// Only setting if nil allows for unit testing with VirtualBoard enabled
 	if drv.config == nil {
-		drv.config = new(device.Config)
-		if err = utilities.MarshalSettings(service.DriverConfigs(), drv.config, true); err != nil {
-			return err
+		drv.svc = &DeviceSDKService{
+			DeviceService: service.RunningService(),
+			lc:            lc,
+		}
+
+		drv.config = &device.ServiceConfig{}
+
+		err := drv.svc.LoadCustomConfig(drv.config, "AppCustom")
+		if err != nil {
+			return errors.Wrap(err, "custom driver configuration failed to load")
+		}
+
+		drv.lc.Debugf("Custom config is : %+v", drv.config)
+
+		err = drv.svc.ListenForCustomConfigChanges(&drv.config.AppCustom, "AppCustom", drv.updateWritableConfig)
+		if err != nil {
+			return errors.Wrap(err, "failed to listen to custom config changes")
 		}
+
```

Comment:
Do we need this config to be "writable"? I don't think so...

Reply:
The 2 timeouts could be modified in real time but it's unlikely
Simulated, pid, and vid require a service reboot to cleanly connect to the device.